### DISCUSSION TITLE
NREL 5MW look-up tables

### DISCUSSION THREAD

**Discussion by MiguelMarante** [NatLabRockies/floris#690] [Q&A] [answered]
Hello,

I have coupled a BEM code (CCBlade) with FLORIS, and I am not sure whether to update the Ct/Cp tables for the NREL 5MW wind turbine.

![image](https://github.com/NREL/floris/assets/133772056/d920032c-825b-428b-a7a3-ee0699da3622)

So I was wondering how these tables were generated and why is the TSR = 8, since it doesn't match with the definition of the NREL 5MW Reference Wind Turbine by Jonkman: https://www.nrel.gov/docs/fy09osti/38060.pdf.

Best regards,
Miguel Pereira

**Comment by rafmudaf** [comment]
@paulf81 do you have any insight?

**Reply by paulf81** [reply, marked as answer]
We didn't generate the Cp/Ct tables ourselves but used existing Cp/Ct tables available for the NREL 5MW.  The TSR value is more a placeholder for possible future usage but is not currently used to modify anything in FLORIS.  We did make a few alterations to the provided Cp/Ct table, generally to increase the smoothness, especially above rated, but these changes were small.  We also limit the Ct to be below 1.0 for numerical reasons (versus what is in the given data).  I found some posts that give Cp/Ct for NREL 5MW for reference:

https://github.com/NREL/turbine-models/blob/master/Offshore/NREL_5MW_126_RWT.csv

But I don't think that is the version I used, it's slightly different, but it will be from a similar source and calculated using openfast.
